custom/icds_reports/utils/aggregation_helpers/aww_incentive.py

```diff
@@ -42,6 +42,7 @@ def aggregation_query(self):
             %(month)s AS month,
             awcm.awc_id,
             awcm.block_id,
+            awcm.supervisor_id,
```

Comment:
you'll need to add this to the group by clause

Reply:
(Done)